### ISSUE TITLE
Add zoomimage compose multiplatform image zoom library

### ISSUE THREAD

**Issue by panpf** [terrakok/Compose-Multiplatform-Wizard#67]
ZoomImage is an gesture zoom viewing of images library specially designed for Compose Multiplatform and Android View. Supported scale, pan, locate, rotation, and super-large image subsampling.

https://github.com/panpf/zoomimage

**Comment by terrakok** [comment]
I agree this is a good library but not for the common wizard. Far not every project needs this
